crates/iota/src/upgrade_compatibility.rs

```diff
@@ -379,19 +399,16 @@ impl CompatibilityMode for CliCompatibilityMode {
             ));
     }
 
-    fn finish(&self, compatibility: &Compatibility) -> Result<(), Self::Error> {
-        let errors: Vec<String> = self
+    fn finish(self, compatibility: &Compatibility) -> Result<(), Self::Error> {
+        let errors: Vec<UpgradeCompatibilityModeError> = self
             .errors
-            .iter()
+            .into_iter()
             .filter(|e| e.breaks_compatibility(compatibility))
             .map(|e| format!("- {:?}", e))
```

Comment:
this line was removed on upstream.

Reply:
There will be 2 more PRs to this branch, it's probably included in them

Reply:
Ah right, sorry missed that, removed it now

Reply:
exactly :D